### ISSUE TITLE
No logging with IntelliJ IDEA 14.1 EAP (IU-140.2683.2)

### ISSUE THREAD

**Issue by talanb** [wakatime/jetbrains-wakatime#23]
When I upgraded the IntelliJ IDEA 14.1 EAP WakaTime seems to have stopped logging data. I see messages for the plugin initialization in the IDEA log, but no data ever gets to my dashboard and no errors are logged. It was working correctly in 14.0.x versions on IntelliJ IDEA.

**Comment by alanhamlett** [comment]
I wasn't able to reproduce this with 14.1 EAP on Mac OS X. Which OS are you using?

**Comment by pawelantczak** [comment]
I can confirm this on Ubuntu 14.04.
.wakatime.cfg exists, but neither IDEA nor Android Studio reports time.

**Comment by pawelantczak** [comment]
Log, anonymized:
```
2015-03-10 20:45:17,066 [ 184625]  DEBUG -                       WakaTime - Executing CLI: [python, /home/.IdeaIC14/config/plugins/WakaTime-resources/wakatime-master/wakatime-cli.py, --key, 95-........-35, --file, /home/ConvertersConfiguration.java, --project, backend, --plugin, Idea/14.0.3 Idea-wakatime/4.0.4] 
2015-03-10 20:45:17,074 [ 184633]  DEBUG -                       WakaTime - python: can't open file '/home/.IdeaIC14/config/plugins/WakaTime-resources/wakatime-master/wakatime-cli.py': [Errno 2] No such file or directory 
2015-03-10 20:45:17,074 [ 184633]  DEBUG -                       WakaTime - Command finished with return value: 2 
```

**Comment by alanhamlett** [comment]
Fixed in v4.0.6 of the plugin. You might need to re-install the WakaTime plugin if it was disabled.
